rpc/handlers.go

```diff
@@ -1149,6 +1152,12 @@ func makeContractError(err error) *jsonrpc.Error {
 	return &contractErr
 }
 
+func makeUnexpectedError(err error) *jsonrpc.Error {
```

Comment:
We could do this for any `*jsonrpc.Error` that needs a data field.

Reply:
See the last commit, let me know what you think.

Reply:
LGTM.